src/gameplay/game.ts

```diff
@@ -594,6 +594,10 @@ Game.prototype.checkBotMoves = function (pendingBots) {
 
   // Players whose moves we're waiting for
   this.interval = setInterval(() => {
+    // hack: check for hammer reject
+    if (thisRoom.howWasWon === 'Hammer rejected.')
+      thisRoom.finished = true;
```

Comment:
Could I ask you to curly bracket this part please? We had a pretty bad issue in the past caused by the lack of curly brackets. Could you also fix up the one in bot.js too?